spec/rspec/mocks/array_including_matcher_spec.rb

```diff
@@ -17,10 +17,6 @@ module ArgumentMatchers
             expect(array_including(1, 2, 3)).to be === [1, 2, 3]
           end
 
-          it "matches the same array,  specified without square brackets" do
```

Comment:
So there is a mistake here, the spec above should be `array_including([1, 2, 3])` and this one left in, if you could change that, that would be grand! (Although you could remove the double space in the title :))